.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# application code
+/securedrop/ @redshiftzero
+/securedrop/ @heartsucker
+
+# ansible / deployment
+/build/         @conorsch
+/devops/        @conorsch
+/install_files/ @conorsch
+/molecule/      @conorsch
+/testinfra/     @conorsch
+/tails_files/   @conorsch
+
+/build/         @msheiny
+/devops/         @msheiny
+/install_files/ @msheiny
+/molecule/      @msheiny
+/tails_files/   @msheiny
+/testinfra/     @msheiny
```

Comment:
Hmm it sounds like from this:

> Order is important. The last matching pattern has the most precedence.

(from [this page](https://github.com/blog/2392-introducing-code-owners)) means that this would result in e.g. only @msheiny being requested on a PR that touches `testinfra` since that is the last matching pattern for `/testinfra/`. Maybe we should put @msheiny and @conorsch on the same line such that they are both requested? 

(Similar comment applies to the entire PR, e.g. the `securedrop/*` section should be `@redshiftzero @dachary @heartsucker` on one line)

Reply:
Ah, you're totally right. Fixing.